Project Management/PI Planning/24PI2 Planning/PI Planning Preparation by Squad/Zowe Explorer Squad - 24PI2 Objectives.md

```diff
@@ -58,6 +58,39 @@
 - *When do you plan to deliver the solution?*
   - 24Q1
 
+### Enable client certificate authentication in Zowe Explorer
+
+- *What problem are you solving?*
+  - Users are unable to configure Zowe Explorer to authenticate to z/OSMF and the Zowe API Mediation Layer using client certificates.
+- *What are you doing to solve it?
+  - Provide a UI that allows users to select certificate files and use those certificates for authentication to z/OSMF or the APIML.
+- *When do you plan to start the work?*
+  - 24Q1
+- *When do you plan to deliver the solution?*
+  - 24Q2
+
+### Option to display Zowe Explorer resources as a table
```

Comment:
Should we leave this ambiguous vs calling out a table.  I think `Allow users to access Zowe Explorer treeview resources in an interactive way` would be good and this will allow us to discuss other possibilities and not be tied to a table view.

Reply:
Sure, makes sense.

Reply:
Updated